ocs_ci/deployment/helpers/tnf_helpers.py

```
@@ -0,0 +1,400 @@
+        ip = next(
+            (addr["address"] for addr in addresses if addr["type"] == "InternalIP"),
+            None,
+        )
+        role = (
+            "master"
+            if "node-role.kubernetes.io/master" in node["metadata"].get("labels", {})
+            else "worker"
+        )
+
+        node_info.append({"name": name, "ip": ip, "role": role})
+        logger.info(f"Node: {name}, IP: {ip}, Role: {role}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)